db/migrate/20230523134506_claim_details.rb

```diff
@@ -7,5 +7,9 @@ def change
     add_column :claims, :preparation_time, :string
     add_column :claims, :time_spent_hours, :integer
     add_column :claims, :time_spent_mins, :integer
+    add_column :claims, :work_before, :string
+    add_column :claims, :work_after, :string
+    add_column :claims, :work_before_date, :date
```

Comment:
given we can determine if this work_before is required based on the presence of the `work_before_date` being populated in the DB. There is a strong argument that we don't need to store `work_before` as a yes no value (same for work_after).

This is the same as the approach for the uplift fields.

Can you look at this and see if you can get it to work.